testing/util/deneb.go

```diff
@@ -78,15 +134,15 @@ func GenerateTestDenebBlockWithSidecar(t *testing.T, parent [32]byte, slot primi
 	sh, err := sbb.Header()
 	require.NoError(t, err)
 	for i, c := range block.Block.Body.BlobKzgCommitments {
-		sidecars[i] = GenerateTestDenebBlobSidecar(t, root, sh, i, c)
+		sidecars[i] = GenerateTestDenebBlobSidecar(t, root, sh, i, c, inclusion[i])
 	}
 
 	rob, err := blocks.NewROBlock(sbb)
 	require.NoError(t, err)
 	return rob, sidecars
 }
 
-func GenerateTestDenebBlobSidecar(t *testing.T, root [32]byte, header *ethpb.SignedBeaconBlockHeader, index int, commitment []byte) blocks.ROBlob {
+func GenerateTestDenebBlobSidecar(t *testing.T, root [32]byte, header *ethpb.SignedBeaconBlockHeader, index int, commitment []byte, incProof [][]byte) blocks.ROBlob {
```

Comment:
My thinking was that if test code uses this func they probably want the flexibility to specify a fake root without building a full block. Otherwise they can use the `GenerateTestDenebBlockWithSidecar` and get the block/header more easily.